docs/using.md

```diff
@@ -48,8 +49,15 @@ for token in md.parse("some *text*"):
 
 The `MarkdownIt` class is instantiated with parsing configuration options,
 dictating the syntax rules and additional options for the parser and renderer.
-You can define this configuration *via* a preset name (`'zero'`, `'commonmark'` or `'default'`),
-or by directly supplying a dictionary.
+You can define this configuration *via* directly supplying a dictionary or a preset name:
+
+- `zero`: This configures the minimum components to parse text (i.e. just paragraphs and text)
+- `commonmark` (default): This configures the parser to strictly comply with the [CommonMark specification](http://spec.commonmark.org/).
+- `js-default`: This is the default in the JavaScript version.
+  Compared to `commonmark`, it disables HTML parsing and enables the table and strikethrough components.
+- `gfm-like`: This configures the parser to approximately comply with the [GitHub Flavored Markdown specification](https://github.github.com/gfm/).
+  Compared to `commonmark`, it disables HTML parsing and enables the table, strikethrough and linkify components.
```

Comment:
Should we add `mdit_py_plugins.tasklists` to this preset? That would make it pretty much GFM spec complete I think.

Reply:
I'd almost go as far as calling the preset just `gfm` instead of `gfm-like` then. What do you think?

Reply:
> Should we add mdit_py_plugins.tasklists to this preset?

No; because (a) the presets do not allow you to load extensions, and anyway (b) the point of moving the plugins to mdit_py_plugins is that markdown-it-py will no longer be dependent on them, in v1.0 mdit_py_plugins will be removed from the `install_requires`.
You could of course add a convenience function(s) in mdit_py_plugins to return a parser setup for full GFM compliance 😄 

> I'd maybe go as far as calling the preset just gfm instead of gfm-like then. What do you think?

No; because then I bet we will have issues opened that "the `gfm` preset behaves differently to when I use Markdown in GitHub". In fact, even with task lists, we are still missing one extension, which is the `tagfilter` extension for raw HTML: https://github.github.com/gfm/#disallowed-raw-html-extension- (currently HTML is just turned off)

On a side note, the good thing with CommonMark, and why I like it as the default, is that there is a comprehensive test suite (https://spec.commonmark.org/) that we test against (tests/test_cmark_spec/commonmark.json) and so can "guarantee" compliance. GFM says it builds in this spec, but the examples given for the extensions in https://github.github.com/gfm are not particularly extensive.

Reply:
I think I'd also enable HTML here, since most HTML is allowed in GFM, just a small subset of tags is disallowed. In fact, "commonmark" could be a better base preset for "gfm-like", as it'd be easy to compare/verify the changes against the annotated parts of the GFM spec.

Reply:
done in bc3a2efae6660a1a27ff28b540d6a7ee2bd7c430